src/libraries/System.Collections/tests/Generic/LinkedList/LinkedList.Generic.Tests.Node.cs

```diff
@@ -52,6 +52,35 @@ public void Verify()
             node.Value = default(T);
 
             VerifyLinkedListNode(node, default(T), null, null, null);
+
+            //[] Verify passing something other then default(T) into the constructor and set the value to something other then default(T)
```

Comment:
This is just a duplicate of the tests covering `Value`, but setting via `ValueRef` instead.